mlir/test/lib/Dialect/SCF/TestLoopParametricTiling.cpp

```diff
@@ -40,6 +40,9 @@ class SimpleParametricLoopTilingPass
   }
 
   void runOnOperation() override {
+    // If no outer loop iteration is given, ignore the pass.
+    if (sizes.empty())
+      return;
```

Comment:
Shouldn't we error out here? Seems like an error from the user to me here, and they would be surprised by the behavior. Any valid workflow where silently ignoring this would be better?

Reply:
@joker-eph Thanks for the feedback. Since it's the invalid input from the user, it should have thrown error instead of ignoring as you suggested. I'll try to change it.